pkg/csi/service/common/commonco/k8sorchestrator/topology.go

```diff
@@ -288,17 +288,18 @@ func azCRAdded(obj interface{}) {
 		log.Errorf("failed to get `name` from AvailabilityZone instance: %+v, Error: %+v", obj, err)
 		return
 	}
-	// Retrieve clusterMoref from instance spec.
-	// TODO: TKGS-HA - convert to slice when appropriate
-	clusterComputeResourceMoId, found, err := unstructured.NestedString(obj.(*unstructured.Unstructured).Object,
-		"spec", "clusterComputeResourceMoId")
+	// Retrieve clusterMorefs from instance spec.
+	clusterComputeResourceMoIds, found, err := unstructured.NestedStringSlice(obj.(*unstructured.Unstructured).Object,
+		"spec", "clusterComputeResourceMoIDs")
 	if !found || err != nil {
```

Comment:
can we add one more check here`len(clusterComputeResourceMoIds) == 0`

Reply:
done